spring-integration-redis/src/test/java/org/springframework/integration/redis/util/RedisLockRegistryTests.java

```diff
@@ -419,6 +421,59 @@ public void testEquals() throws Exception {
 		lock2.unlock();
```

Comment:
From RedisLockRegistry title javadoc: However, locks are scoped by the registry; a lock from a different registry with the same key (even if the registry uses the same 'registryKey') are different locks, and the second cannot be acquired by the same thread while the first is locked.